### PR TITLE
Fix Dell C6320 'TotalSystemMemoryGiB' is 0.25

### PR DESCRIPTION
**Pull request by zyoung51**
Resolves ODR-792

### CODE REVIEW

File: data/templates/redfish.1.0.0.computersystem.1.0.0.json [hunk 1]
```diff
@@ -28,7 +28,9 @@
         <% var memsize = 0; %>
         <% dmi.data['Memory Device'].forEach(function(dev) { %>
         <%     var size = dev.Size.split(' ')[0]; %>
+        <%     var units = dev.Size.split(' ')[1]; %>
         <%     size = isNaN(size) ? 0 : size; %>
+        <%     size = (units === 'GB') ? size * 1024 : size; %>
```
Comment (yyscamper) [inline data/templates/redfish.1.0.0.computersystem.1.0.0.json]: How about `MB`?

**Comment by yyscamper** [comment]
:+1:

**Comment by jlongever** [comment]
test this please

**Comment by yyscamper** [comment]
test this please

**Comment by yyscamper** [comment]
test this please

**Pull request merged**